src/App.css

```
@@ -923,6 +1013,496 @@ table.not-prose th {
+  cursor: ns-resize;
+}
+
+.notion-table-edge-resize.is-column {
+  cursor: ew-resize;
+}
+
+.notion-table-edge-resize::before {
+  content: "";
+  width: 12px;
+  height: 4px;
+  border-block: 1px solid currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)